docs/Inputs.md

```diff
@@ -324,7 +324,7 @@ If the default Dropzone label don't fit with your need, you can pass a `placehol
 </ImageInput>
 ```
 
-Note that the image upload returns a [File](https://developer.mozilla.org/en/docs/Web/API/File) object. It is your responsibility to handle it depending on your API behavior. You can for instance encode it in base64, or send it as a multi-part form data.
+Note that the image upload returns a [File](https://developer.mozilla.org/en/docs/Web/API/File) object. It is your responsibility to handle it depending on your API behavior. You can for instance encode it in base64, or send it as a multi-part form data. Check [here](./RestClients.html#decorating-your-rest-client-example-of-file-upload) for an example to do base64 encoding by extending the REST Client.
```

Comment:
I don't like "here" link. Please add a link on a description of what the reader will see when they click on the link.